dubbo-common/src/main/java/org/apache/dubbo/common/config/PropertiesConfiguration.java

```diff
@@ -53,6 +53,9 @@ public PropertiesConfiguration() {
         //override the properties.
         for (OrderedPropertiesProvider orderedPropertiesProvider :
                 orderedPropertiesProviders) {
+            if (orderedPropertiesProvider == null) {
```

Comment:
Maybe this change off the point. I think change the code here will be better.

https://github.com/apache/dubbo/blob/f26ba91b67f642148a10d3b197502e29928b77bf/dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java#L425-L428

->
```
    public T getExtension(String name) {
        T extension = getExtension(name, true);
        if (extension == null) {
            throw new IllegalArgumentException("Not find extension: " + name);
        }
        return extension;
    }
```